test/topology/test_topology.py

```diff
@@ -90,6 +93,24 @@ async def no_joining_nodes():
     await table.add_column()
     await random_tables.verify_schema()
 
+@pytest.mark.asyncio
+@pytest.mark.skip(reason="Replace operation sleeps for 60 seconds")
+async def test_replace_different_ip(manager: ManagerClient, random_tables) -> None:
+    servers = await manager.running_servers()
+    await manager.server_stop(servers[0].server_id)
+    replace_cfg = ReplaceConfig(replaced_id = servers[0].server_id, reuse_ip_addr = False)
+    await manager.server_add(replace_cfg)
+    # TODO: check that group 0 no longer contains the replaced node
+
+@pytest.mark.asyncio
+@pytest.mark.skip(reason="As above + the new node cannot join group 0")
```

Comment:
Nitpick: if this is a bug, we should open an issue for it - and then can refer to that issue here in the skip reason.

Reply:
The situation is dynamic, with Kostja's dynamic IP change support PR (https://github.com/scylladb/scylladb/pull/12035) the new node does manage to join group 0, however there's another problem that comes up later during the procedure.